src/lib/y2partitioner/widgets/description_section/blk_device.rb

```diff
@@ -54,11 +54,14 @@ def entries
 
         # Entry data about the device name
         #
+        # It could include complementary data (e.g., "(Ext4 Journal)")
+        #
+        # @see #journal_info
         # @see BlkDeviceAttributes
         #
         # @return [String]
         def device_value
-          device_name
+          [device_name, journal_info].compact.join(" ")
```

Comment:
As commented in janghouts, I think this took a bad decision here. It would be better to have an new entry in the filesystem description:

Journal device: /dev/sda1